ngraph/frontend/onnx/frontend/src/place.cpp

```diff
@@ -132,3 +157,14 @@ bool PlaceTensorONNX::is_equal(Place::Ptr another) const
     }
     return false;
 }
+
+bool PlaceTensorONNX::is_equal_data(Place::Ptr another) const
+{
+    const auto consuming_ports = get_consuming_ports();
+    bool eq_to_consuming_port =
+        std::any_of(consuming_ports.begin(), consuming_ports.end(), [&another](const Ptr& place) {
+            return place->is_equal(another);
+        });
+    return is_equal(another) || eq_to_consuming_port ||
```

Comment:
Not a big deal, but if you check `eq_to_consuming_port` first and it's equal to true, you won't have to call `is_equal(another)` which could potentially be false and waste cpu cycles.

Reply:
right, changed to lambda call